guides/embedders/mistral.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Semantic Search with Mistral Embedding - Meilisearch documentation
+description: This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities.
+---
+
+# Semantic Search with Mistral Embedding
+
+## Introduction
+
+This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities. By leveraging Meilisearch's AI features and Mistral's embedding API, you can enhance your search experience and retrieve more relevant results.
+
+## Requirements
+
+To follow this guide, you'll need:
+
+- A Meilisearch Cloud project running version 1.6 or above with the Vector store activated.
+- A Mistral account with an API key for embedding generation. You can sign up for a Mistral account at [Mistral](https://mistral.ai/).
+- No backend required.
+
+## Explanation
+
+Meilisearch has the ability to automatically generate embeddings for your documents by configuring an embedder within the engine. You can use a local embedder, which may be slower depending on your cloud account, or leverage a third-party API. The third-party API can be a self-hosted API based on Ollama or a service using the dedicated REST option.
+
+By utilizing this configuration, Meilisearch will automatically generate embeddings for your documents based on the template you've set up (optional but recommended) and store the embeddings in the vector store. Meilisearch optimizes the calls to the third-party API by auto-batching the requests. Additionally, Meilisearch uses this configuration to automatically generate embeddings for each search query, which are then used to query the Approximate Nearest Neighbor (ANN) in the vector storage.
+
+## Setting up Meilisearch
+
+To set up an embedder in Meilisearch, you simply need to add it to your settings. You can refer to the [Meilisearch documentation](https://www.meilisearch.com/docs/reference/api/settings#update-embedder-settings) for more details on updating the embedder settings.
+
+Here's an example configuration for using Mistral embedding:
+
+```json
+{
+  "mistral": {
+    "source": "rest",
+    "apiKey": "<Mistral API Key>",
+    "dimensions": 1024,
+    "documentTemplate": "<Custom template (Optional)>",
+    "url": "https://api.mistral.ai/v1/embeddings",
+    "request": {
+      "model": "mistral-embed",
```

Comment:
Earlier, you should say that you aim at using the `mistral-embed` model, so that this instruction makes sense.

Alternatively, add comments in the JSON to explain this field